### PR TITLE
Update readme and contribution guidelines

### PR DESCRIPTION
**Pull request by nCastle1**
This new format is intended to tie the various aspects of the .NET SDK together into a cohesive whole. Additionally, these changes make the landing pages more visual and accessible, with a goal of better showcasing our excellent SDK resources.

Contributing.md is updated as well to match the readme and license.

Note to reviewers: This PR should not be merged before #405

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+8. Self-review your code.
+9. Test your code.
 
 ### Submitting changes
+
 1. Push the changes in your feature branch to your repo.
-2. Submit a [pull request](https://help.github.com/articles/using-pull-requests) against the "main" branch. 
-3. In the pull request comments, add a reference to the issue the pull request resolves.  For example, if the pull request resolves issue 45, add a comment to the pull request that includes "#45" (no quotes).    
+2. Submit a [pull request](https://help.github.com/articles/using-pull-requests) against the "main" branch.
+3. In the pull request comments, add a reference to the issue the pull request resolves. For example, if the pull request resolves issue 45, add a comment to the pull request that includes "#45" (no quotes).
```
Comment (ThadT) [inline CONTRIBUTING.md:34]: I think you could format this as "code" so you wouldn't have to use quotes (?).
```suggestion
3. In the pull request comments, add a reference to the issue the pull request resolves. For example, if the pull request resolves issue 45, add a comment to the pull request that includes `#45`.
```

File: README.md [hunk 2]
```diff
@@ -1,66 +1,77 @@
...
+[![Link: Documentation](https://img.shields.io/badge/Documentation-633b9b?style=flat-square)](https://developers.arcgis.com/net/)
+[![Link: Tutorials](https://img.shields.io/badge/Tutorials-633b9b?style=flat-square)](https://developers.arcgis.com/documentation/mapping-apis-and-services/tutorials/)
+[![Link: Samples](https://img.shields.io/badge/Samples-633b9b?style=flat-square)](https://developers.arcgis.com/net/wpf/sample-code/)
+[![Link: Demos](https://img.shields.io/badge/Demos-633b9b?style=flat-square)](https://github.com/Esri/arcgis-runtime-demos-dotnet)
+![Badge: Toolkit](https://img.shields.io/badge/Toolkit-black?style=flat-square)
+[![Link: Templates](https://img.shields.io/badge/Templates-633b9b?style=flat-square&logo=visualstudio&labelColor=gray)](https://github.com/Esri/arcgis-runtime-templates-dotnet)
+[![Link: NuGet](https://img.shields.io/badge/NuGet-633b9b?style=flat-square&logo=nuget&labelColor=gray)](https://www.nuget.org/profiles/Esri_Inc)
+[![Link: Esri Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Esri%20Community-633b9b?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-net/bd-p/arcgis-runtime-sdk-dotnet-questions)
+
```
Comment (ThadT) [inline README.md:10]: Love the badges 👏

File: README.md [hunk 3]
```diff
@@ -1,66 +1,77 @@
...
 # ArcGIS Runtime SDK for .NET - Toolkit
 
-[![doc](https://img.shields.io/badge/Doc-purple)](https://esri.github.io/arcgis-toolkit-dotnet/controls.html) [![API reference](https://img.shields.io/badge/API_Ref-Android-lime)](https://esri.github.io/arcgis-toolkit-dotnet/api/android/index.html) [![API reference](https://img.shields.io/badge/API_Ref-iOS-darkblue)](https://esri.github.io/arcgis-toolkit-dotnet/api/ios/index.html) [![API reference](https://img.shields.io/badge/API_Ref-UWP-skyblue)](https://esri.github.io/arcgis-toolkit-dotnet/api/uwp/index.html) [![API reference](https://img.shields.io/badge/API_Ref-Forms-blue)](https://esri.github.io/arcgis-toolkit-dotnet/api/netstd/index.html) [![API reference](https://img.shields.io/badge/API_Ref-WPF-blueviolet)](https://esri.github.io/arcgis-toolkit-dotnet/api/netfx/index.html)
+The ArcGIS Runtime SDK for .NET Toolkit contains controls and components you can use to accelerate your development with [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
```
Comment (ThadT) [inline README.md:13]:
```suggestion
The ArcGIS Runtime SDK for .NET Toolkit contains controls and components to accelerate your development with [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
```

File: README.md [hunk 4]
```diff
@@ -1,66 +1,77 @@
...
-- **SymbolDisplay**: Renders a symbol in a control.
-- **TimeSlider**: Allows interactively defining a temporal range (i.e. time extent) and animating time moving forward or backward.  Can be used to manipulate the time extent in a MapView or SceneView.
-
-### Features in Preview
+|Component |UWP  |WPF  |Xamarin.Android  |Xamarin.iOS  |Xamarin.Forms |
+|---|---|---|---|---|---|
+|[ARSceneView](https://esri.github.io/arcgis-toolkit-dotnet/ar.html)   | N/A | N/A | ✔ | ✔ | ✔ |
+|[BasemapGallery](https://esri.github.io/arcgis-toolkit-dotnet/basemap-gallery.html) | ✔ | ✔ | ❌ | ❌ | ✔ |
+|[BookmarksView](https://esri.github.io/arcgis-toolkit-dotnet/bookmarks-view.html)   | ✔ | ✔ | ✔ | ✔ | ✔ |
+|Compass   | ✔ | ✔ | ✔ | ✔ | ✔ |
```
Comment (ThadT) [inline README.md:33]: Can we add doc links to all components in the table?

File: README.md [hunk 5]
```diff
@@ -1,66 +1,77 @@
...
+[![Link: Toolkit API Reference - WPF](https://img.shields.io/badge/WPF-0078d6?style=flat-square&labelColor=gray&logo=windowsxp)](https://esri.github.io/arcgis-toolkit-dotnet/api/netfx/index.html)
+[![Link: Toolkit API Reference - UWP](https://img.shields.io/badge/UWP-0078d6?style=flat-square&labelColor=gray&logo=windows)](https://esri.github.io/arcgis-toolkit-dotnet/api/uwp/index.html)
 
-There are two ways to add Toolkit to your project:
+| <img width="150" alt="Basemap Gallery" src="https://user-images.githubusercontent.com/29742178/124198151-f2dc6380-da84-11eb-8e78-4e705d14c33d.png" /> | <img alt="Search View" width="150" src="https://user-images.githubusercontent.com/29742178/142301018-4bbeb0f2-3021-49a7-b5ec-f642c5700bd0.png" /> | <img src="https://user-images.githubusercontent.com/29742178/121975740-34f07000-cd37-11eb-9162-462925cb3fe7.png" width="150" alt="Overview Map" /> | <img src="https://user-images.githubusercontent.com/1378165/73389924-011fc200-428a-11ea-91bf-4ea1c2bf6683.png" width="105" alt="Legend" /> |
```
Comment (ThadT) [inline README.md:23]: A `title` attribute would be good to show the component name when mousing over an image (same text as `alt`?).
```suggestion
| <img width="150" alt="Basemap Gallery" title="Basemap Gallery" src="https://user-images.githubusercontent.com/29742178/124198151-f2dc6380-da84-11eb-8e78-4e705d14c33d.png" /> | <img alt="Search View" title="Search View" width="150" src="https://user-images.githubusercontent.com/29742178/142301018-4bbeb0f2-3021-49a7-b5ec-f642c5700bd0.png" /> | <img src="https://user-images.githubusercontent.com/29742178/121975740-34f07000-cd37-11eb-9162-462925cb3fe7.png" width="150" alt="Overview Map" title="Overview Map"/> | <img src="https://user-images.githubusercontent.com/1378165/73389924-011fc200-428a-11ea-91bf-4ea1c2bf6683.png" width="105" alt="Legend" title="Legend"/> |
```

File: README.md [hunk 6]
```diff
@@ -1,66 +1,77 @@
...
+[![Link: Toolkit NuGet - Xamarin.Forms](https://img.shields.io/badge/Xamarin.Forms%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms/)
+[![Link: Toolkit NuGet - AR](https://img.shields.io/badge/Augmented%20Reality%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit/)
+[![Link: Toolkit NuGet - Forms AR](https://img.shields.io/badge/Xamarin.Forms%20Augmented%20Reality%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit.Forms/)
 
-## Issues
+## Customize
 
-Find a bug or want to request a new feature? Please let us know by [submitting an issue](https://github.com/Esri/arcgis-toolkit-dotnet/issues/new).
+Toolkit is provided as an open source project to enable you to customize for your requirements. [See the docs](https://esri.github.io/arcgis-toolkit-dotnet/buildingtoolkit.html) for instructions on building the Toolkit yourself.
```
Comment (ThadT) [inline README.md:58]:
```suggestion
The ArcGIS Runtime SDK Toolkit is provided as an open-source project so you can customize it for your requirements. [See the docs](https://esri.github.io/arcgis-toolkit-dotnet/buildingtoolkit.html) for instructions on building the Toolkit yourself.
```

**Review by ThadT — APPROVED** [review]
Looks really good, @nCastle1. I just have a couple small suggestions.

File: README.md (hunk 4 above)
Comment (nCastle1) [reply README.md:47]: I would like to, but unfortunately we don't have guide doc for most components. I've been back filling doc as I have time.

**Review by dotMorten — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1,66 +1,93 @@
...
+[![Link: Templates](https://img.shields.io/badge/Templates-633b9b?style=flat-square&logo=visualstudio&labelColor=gray)](https://github.com/Esri/arcgis-runtime-templates-dotnet)
+[![Link: NuGet](https://img.shields.io/badge/NuGet-633b9b?style=flat-square&logo=nuget&labelColor=gray)](https://www.nuget.org/profiles/Esri_Inc)
+[![Link: Esri Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Esri%20Community-633b9b?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-net/bd-p/arcgis-runtime-sdk-dotnet-questions)
 
-The ArcGIS Runtime SDK for .NET Toolkit contains controls and utilities you can use with [ArcGIS Runtime SDK for .NET](http://links.esri.com/dotnetsdk).
+# ArcGIS Runtime SDK for .NET - Toolkit
 
-There are two ways to add Toolkit to your project:
+The ArcGIS Runtime SDK for .NET Toolkit contains controls and components to accelerate your development with [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
+
```
Comment (mbranscomb) [inline README.md:14]:
```suggestion
## Documentation

```

File: README.md [hunk 8]
```diff
@@ -1,66 +1,93 @@
...
+</table>
 
-1. **Install from Nuget** - the fastest way to get toolkit into your app
-    - [Esri.ArcGISRuntime.Toolkit](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit)
-	- [Esri.ArcGISRuntime.Toolkit.Xamarin.Forms](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms)
-	- [Esri.ArcGISRuntime.ARToolkit](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit)
-	- [Esri.ArcGISRuntime.ARToolkit.Forms](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit.Forms)
-2. **[Build from source](https://esri.github.io/arcgis-toolkit-dotnet/buildingtoolkit.html)** - do this if you want to customize toolkit
-   
 ## Features
```
Comment (mbranscomb) [inline README.md:41]: How about moving the feature availability table to a separate section in the readme (or even a different doc) and instead include some brief doc about the features, including screenshots?

File: README.md [hunk 9]
```diff
@@ -1,66 +1,93 @@
...
+[![Link: Toolkit API Reference - Android](https://img.shields.io/badge/Xamarin.Android-3ddc84?style=flat-square&labelColor=gray&logo=android)](https://esri.github.io/arcgis-toolkit-dotnet/api/android/index.html)
+[![Link: Toolkit API Reference - iOS](https://img.shields.io/badge/Xamarin.iOS-black?style=flat-square&labelColor=gray&logo=ios)](https://esri.github.io/arcgis-toolkit-dotnet/api/ios/index.html)
+[![Link: Toolkit API Reference - Forms](https://img.shields.io/badge/Xamarin.Forms-3498db?style=flat-square&labelColor=gray&logo=Xamarin)](https://esri.github.io/arcgis-toolkit-dotnet/api/netstd/index.html)
+[![Link: Toolkit API Reference - WPF](https://img.shields.io/badge/WPF-0078d6?style=flat-square&labelColor=gray&logo=windowsxp)](https://esri.github.io/arcgis-toolkit-dotnet/api/netfx/index.html)
+[![Link: Toolkit API Reference - UWP](https://img.shields.io/badge/UWP-0078d6?style=flat-square&labelColor=gray&logo=windows)](https://esri.github.io/arcgis-toolkit-dotnet/api/uwp/index.html)
+
+<table aria-hidden="true">
```
Comment (mbranscomb) [inline README.md:23]: Consider moving the screenshots to the Features section?

**Review by mbranscomb — COMMENTED** [review]
@nCastle1 Much improved! Added a couple of suggestions.

**Review by mbranscomb — APPROVED** [review]
@nCastle1 Looks great, nice work.

**Pull request merged**